cog/core/downtime.py

```
@@ -0,0 +1,85 @@
+
+
+@dataclass
+class Downtime:
+    start: datetime
+    # TODO: will Downtime.end be None?
+    end: datetime = field(default_factory=datetime.now)
+    is_restored: bool = False
+
+    def __contains__(self, timestamp: datetime):
+        start = self.start
+        end = self.end or datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)